internals/daemon/daemon.go

```diff
@@ -28,9 +28,9 @@ import (
 	"runtime"
 	"strings"
 	"sync"
-	"syscall"
 	"time"
 
+	"golang.org/x/sys/unix"
```

Comment:
Why the change from syscall into an external package?

Reply:
The `syscall` package is deprecated. See https://groups.google.com/g/golang-dev/c/cEASnHIXmLI/m/WUThor2mBwAJ

Reply:
I've decided to revert to syscall, as the entire Pebble is using this package. I think a clean migration as a separate exercise would be better, once we are ready to migrate.